rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

```diff
@@ -2,6 +2,31 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[july-5-2024]]
+== July 5, 2024
+
+// == What's New
+//include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
```

Comment:
Do we now include the What's New heading even if there aren't any new features? I haven't done this in the past, and don't see the guidance for that in the standards page, although I could be missing it. 
https://confluence.internal.salesforce.com/display/MTDT/Fixed+Issue+Descriptions

Reply:
The header is commented out, so it won't show.